Prototyper/Classes/Feedback/FeedbackView.swift

```diff
@@ -0,0 +1,121 @@
+//
+//  FeedbackView.swift
+//  KeychainSwift
+//
+//  Created by Raymond Pinto on 20.12.19.
+//
+
+import SwiftUI
+
+struct FeedbackView: View {
+    @EnvironmentObject var model: Model
+    @State var descriptionText: String = ""
+    @State var showSheet: Bool = false
+    @State var showScreenshot: Bool = true
+    @State var activeSheet: ActiveSheet = .loginSheet
+    @State var showSendFeedbackView: Bool = false
+    @State var feedback: Feedback?
+    
+    enum ActiveSheet {
+        case loginSheet, MarkupSheet
+    }
+    
+    var body: some View {
+        NavigationView {
+            VStack {
+                HStack (alignment: .top) {
+                    if showScreenshot {
+                        ZStack (alignment: .topTrailing) {
+                            ZStack {
+                                Image(uiImage: model.screenshot)
+                                    .resizable()
+                                    .border(Color(white: 0.75))
```

Comment:
I would suggest to use a shadow that is similar to the one found in the suggestion in the edit view. This makes it look a bit flashier then the simple border and still differentiates it from the background.